packages/core/src/textures/BaseTexture.js

```diff
@@ -136,12 +136,12 @@ export class BaseTexture extends EventEmitter
         this.target = target || TARGETS.TEXTURE_2D;
 
         /**
-         * Set to true to enable pre-multiplied alpha
+         * How to treat premultiplied alpha, see {@link PIXI.PMA_MODES}.
          *
-         * @member {boolean}
-         * @default true
+         * @member {PIXI.PMA_MODES}
+         * @default PIXI.PMA_MODES.DO_UNPACK
          */
-        this.premultiplyAlpha = premultiplyAlpha !== false;
+        this.premultiplyAlpha = premultiplyAlpha !== undefined ? Number(premultiplyAlpha) : PMA_MODES.DO_UNPACK;
```

Comment:
We could be doing a better job here of signaling to users to convert their code to use PMA_MODES instead of using the boolean.
```suggestion
        if (typeof premultiplyAlpha === 'boolean')
        {
            deprecation('v5.2.0', 'premultiplyAlpha is not a boolean anymore, see PIXI.PMA_MODES`);
            premultiplyAlpha = Number(premultiplyAlpha);
        }
        this.premultiplyAlpha = premultiplyAlpha !== undefined ? premultiplyAlpha : PMA_MODES.DO_UNPACK;
```

Reply:
Don't merge my suggestion, you'll probably need to import `deprecation` from `@pixi/utils` and make a better message

Reply:
Actually we have even bigger problem. I posted solutions like `baseTexture.premultiplyAlpha=false` several times. and now it wont work because its a field, we can detect that change and all `===` statements will ignore that. Any ideas?

Reply:
I agree on deprecations.

Reply:
I wont merge this of course, because you have even more errors there :) like two different quotation symbols

Reply:
yeah, we have other modes. I agree, we need to rename it.